Userland/Applications/Presenter/PresenterWidget.cpp

```diff
@@ -34,9 +36,16 @@ ErrorOr<void> PresenterWidget::initialize_menubar()
             return;
         this->set_file(response.value()->filename());
     });
+
+    m_settings_window = TRY(GUI::SettingsWindow::create("Presenter Settings"));
+    (void)TRY(m_settings_window->add_tab<PresenterSettingsFooterWidget>("Footer", "footer"sv));
+    auto settings_action = GUI::Action::create("&Settings...", [this](auto&) {
```

Comment:
**moar** icons 📈. also, we don't seem to be using ellipsis for the settings action across the system (c19486172d7d6183ee7c25c1c95d58eba765eb48).

```suggestion
    m_settings_window = TRY(GUI::SettingsWindow::create("Presenter Settings"));
    m_settings_window->set_icon(window->icon());
    (void)TRY(m_settings_window->add_tab<PresenterSettingsFooterWidget>("Footer", "footer"sv));
    auto settings_action = GUI::Action::create("&Settings", TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/settings.png"sv)), [this](auto&) {
```